allennlp/training/trainer_base.py

```diff
@@ -36,12 +36,18 @@ def __init__(
         rank: int = 0,
```

Comment:
Type of cuda_device a few lines up should be just `int` no`.

Reply:
Thanks, fixed